.githooks/setDefaultHookPath.sh

```diff
@@ -0,0 +1,10 @@
+# We cannot modify your .git folder where the hooks are stored in
```

Comment:
the file `setDefaultHookPath.sh` is not executable directly after pulling the file (a `chmod +x` on the file is needed). i'd suggest to check in an executable version of the file to avoid errors